docs/src/main/resources/microsite/js/sponsors.js

```diff
@@ -0,0 +1,96 @@
+function addSponsor(divId, member, size) {
+    div = document.getElementById(divId);
+    var a = document.createElement('a');
+    a.setAttribute('href', member.website || member.profile || '#');
+    var img = document.createElement('img');
+    img.setAttribute('src', member.image || 'img/missing-avatar.svg');
+    if (size) {
+        img.setAttribute('height', size);
+        img.setAttribute('width', size);
+    }
+    img.setAttribute('alt', member.name);
+    img.setAttribute('style', 'margin:6px;');
+    if (member.marginBottom)
+        img.setAttribute('style', img.getAttribute('style') + 'margin-bottom:' + member.marginBottom + 'px;')
+    a.appendChild(img);
+    div.appendChild(a);
+};
+
+const PlatinumSize = 98;
+const GoldSize = 76;
+const SilverSize = 60;
+const BackerSize = 45;
+const ContributorSize = 36;
+
+var sponsors = function() {
+    var xhr = new XMLHttpRequest();
+    xhr.open('GET', 'https://opencollective.com/typelevel/members/all.json', true);
+    xhr.responseType = 'json';
+    xhr.onload = function() {
+        var status = xhr.status;
+        if (status === 200) {
+            for(i = 0; i < xhr.response.length; i++) {
+                var member = xhr.response[i];
+                if (member.isActive) {
+                    switch (member.tier) {
+                    case 'Platinum Sponsor':
+                        addSponsor('platinum-sponsors', member, PlatinumSize);
+                    case 'Gold Sponsor':
+                        addSponsor('gold-sponsors', member, GoldSize);
+                    case 'Silver Sponsor':
+                        addSponsor('silver-sponsors', member, SilverSize);
+                    case 'backer':
+                        addSponsor('backers', member, BackerSize);
+                        break;
+                    default:
+                        if (member.totalAmountDonated > 0) {
+                            addSponsor('other-contributors', member, ContributorSize);
+                        }
+                    }
+                };
+            }
+        }
+    };
+    xhr.send();
+};
```

Comment:
NBD, but [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) is the modern way. Something like this I think:

```suggestion
var sponsors = async function() {
    var response = await fetch('https://opencollective.com/typelevel/members/all.json');
    if (response.ok) {
        var members = await response.json();
        for(i = 0; i < members.length; i++) {
            var member = members[i];
            if (member.isActive) {
                switch (member.tier) {
                case 'Platinum Sponsor':
                    addSponsor('platinum-sponsors', member, PlatinumSize);
                case 'Gold Sponsor':
                    addSponsor('gold-sponsors', member, GoldSize);
                case 'Silver Sponsor':
                    addSponsor('silver-sponsors', member, SilverSize);
                case 'backer':
                    addSponsor('backers', member, BackerSize);
                    break;
                default:
                    if (member.totalAmountDonated > 0) {
                        addSponsor('other-contributors', member, ContributorSize);
                    }
                }
            }
        }
    }
};
```

Reply:
Thanks. I don't know and I don't have time to retest it, but if a knowledgeable reviewer agrees, please feel free to commit it!

Reply:
If it ain't broke don't fix it! 😆

Reply:
Lets move forward with this, and then we can PR this in a later review. We want to fix whats here most importantly.

Reply:
:+1:  My CSS is dated and my JS was never competent, and would be delighted to see either improved once the fire is out.